cmd/osv-scanner/fix/noninteractive.go

```diff
@@ -423,7 +423,7 @@ func makeResultVuln(vuln resolution.Vulnerability) vulnOutput {
 		vk := sg.Nodes[sg.Dependency].Version
 		affected[packageOutput{Name: vk.Name, Version: vk.Version}] = struct{}{}
 	}
-	v.Packages = slices.Collect(maps.Keys(affected))
+	v.Packages = slices.AppendSeq(make([]packageOutput, 0, len(affected)), maps.Keys(affected))
```

Comment:
Is this needed? It seems weird the .Collect doesn't reallocate slices. 

Even if it does not, I think it is better to stick with Collect here until we see that this causes a performance problem just for code clarity

Reply:
Apparently [using `slices.AppendSeq` is the "simplest true equivalent"](https://github.com/golang/exp/blob/939b2ce775acaac3abfa605a85c57e43fc448693/maps/maps.go#L20), and there is a [performance difference](https://github.com/ldez/exptostd/issues/9).

It doesn't look like there's much difference between `slices.Collect` though, so I'm not fussed - I can do some benchmarking later too